thunder/executors/cudnnex.py

```diff
@@ -124,23 +116,26 @@ def __setitem__(self, key, value):
 _cudnnex_cache = CudnnexLRUCache(maxlen=1024)
 
 
-def _make_cudnn_sdpa_forward_graph(query, key, value, attn_mask, dropout_p, is_causal):
+def _make_cudnn_sdpa_forward_graph(
+    query, key, value, attn_mask, dropout_p, is_causal, query_stride, key_stride, value_stride
```

Comment:
The main change here is that both fwd and bwd graph take in explicit input strides.